app/views/components/show_file_table.scala.html

```diff
@@ -15,14 +15,15 @@
  *@
 
 @import views.ViewUtils._
+@import viewmodels.FileView
 
-@(files: Seq[FileAttachment], mode: Mode)(implicit messages: Messages)
+@(files: Seq[FileView], mode: Mode)(implicit messages: Messages)
 
 <dl class="file-list mb-5">
-    @for((file: FileAttachment, index: Int) <- files.zipWithIndex) {
+    @for((file: FileView, index: Int) <- files.zipWithIndex) {
         <div id="files-row-@{index}" class="file-list-row">
             <dt class="file-list-name" id="files-row-@{index}-title">@file.name</dt>
-            <dd class="file-list-size" id="files-row-@{index}-size" >@humanReadableSize(file.size)</dd>
+            <dd class="file-list-size" id="files-row-@{index}-size" >@confidentialityStatusLabel(file.confidential)</dd>
```

Comment:
Should we update this class and id from `size` to something else more representative?